Makefile

```diff
@@ -76,3 +77,8 @@ deploy: manifests ## Deploy controller to the K8s cluster specified in ~/.kube/c
 
 undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/config.
 	$(KUSTOMIZE) build config/default | $(KO) delete -f -
+
+##@ Publish
+
+publish: ## Publish OCI image and prepare Kubernetes resources for release
```

Comment:
Not sure if the publish target has much use outside the CI. Let me know if we should move this command to the CI configuration.

Reply:
I had the same thought. The difference between publish and deploy is that publish doesn't apply the config to the cluster.